src/ports/postgres/modules/assoc_rules/assoc_rules.sql_in

```diff
@@ -579,7 +579,7 @@ AS $$
                                        input_table,
                                        output_schema,
                                        verbose,
-                                       'NULL');
+                                       10);
 $$ LANGUAGE plpythonu
```

Comment:
The `if-elif` in https://github.com/apache/madlib/blob/master/src/ports/postgres/modules/assoc_rules/assoc_rules.py_in#L66 can be simplified to a single `if` statement due to this change, can you please update that code too?

Reply:
changed it. Thanks for pointing it out.